Cargo.toml

```diff
@@ -23,6 +23,7 @@ async-trait = { version = "0.1.57", default-features = false }
 bytes = { version = "1.2.1", default-features = false, features = ["std", "serde"] }
 clap = { version = "4", features = ["std", "color", "usage", "help", "error-context", "suggestions", "derive", "env"], optional = true }
 color-eyre = { version = "0.6.2", default-features = false, features = [ "track-caller", "issue-url", "tracing-error", "capture-spantrace", "color-spantrace" ], optional = true }
+detsys-ids-client = { git = "https://github.com/DeterminateSystems/detsys-ids-client.git", optional = true }
```

Comment:
Do we want to publish this crate and instead version it so we can continue to release on crates.io? If not, let's remove the "publish on crates.io" from the release process docs (even though we ourselves haven't been doing a great job of doing that anyways).